erpnext/patches/v13_0/validate_options_for_data_field.py

```diff
@@ -9,18 +9,18 @@
 def execute():
 
     for field in frappe.get_all('Custom Field',
-                            fields = ['name'],
+                            fields = ['name', 'options'],
                             filters = {
                                 'fieldtype': 'Data',
-                                'options': ['!=', None]
+                                'options': ['is', 'set']
                             }):
 
-        if field not in data_field_options:
-            frappe.db.sql("""
-                UPDATE
-                    `tabCustom Field`
-                SET
-                    options=NULL
-                WHERE
-                    name=%s
-            """, (field))
+        if field.options not in data_field_options:
+            custom_field = frappe.qb.DocType('Custom Field')
+            frappe.qb.update(
+                custom_field
+            ).set(
+                custom_field.options, "NULL"
```

Comment:
this is the string `"null"` and not SQL `NULL`. 😄 

It should be `None` IIRC

Reply:
also this whole patch doesn't belong in ERPNext. Framework added this validation, so patch/fix should be in framework.

Reply:
Yeah thanks for pointing this out!
Will close this one and raise a PR in FF

Reply:
instead of closing this, reuse it to remove that framework patch? 😅